static/app/components/core/badge/featureBadge.tsx

```diff
@@ -1,34 +1,16 @@
-import {Fragment, type ReactNode} from 'react';
 import {useTheme} from '@emotion/react';
 import styled from '@emotion/styled';
 import type {SeverityLevel} from '@sentry/core';
 import {captureException, withScope} from '@sentry/react';
 
 import CircleIndicator from 'sentry/components/circleIndicator';
-import Badge, {type BadgeType} from 'sentry/components/core/badge';
+import Badge from 'sentry/components/core/badge';
 import type {TooltipProps} from 'sentry/components/tooltip';
 import {Tooltip} from 'sentry/components/tooltip';
 import {t} from 'sentry/locale';
-import type {ValidSize} from 'sentry/styles/space';
 import {space} from 'sentry/styles/space';
 
-export type FeatureBadgeType = Extract<
-  BadgeType,
-  'alpha' | 'beta' | 'new' | 'experimental' | 'internal'
->;
-
-type BadgeProps = {
-  type: FeatureBadgeType;
-  condensed?: boolean;
-  expiresAt?: Date;
-  title?: ReactNode;
-  tooltipProps?: Partial<TooltipProps>;
-  variant?: 'badge' | 'indicator' | 'short';
-};
-
-type Props = Omit<React.HTMLAttributes<HTMLDivElement>, keyof BadgeProps> & BadgeProps;
```

Comment:
This is a a major code smell. I refactored to use interfaces and noticed that it conflicts with HTML title attribute.